recipes/slamdunk/meta.yaml

```diff
@@ -14,23 +14,27 @@ build:
     - slamdunk=slamdunk.slamdunk:run
     - alleyoop=slamdunk.alleyoop:run
     - splash=slamdunk.splash:run
-  number: 0
   script: {{ PYTHON }} -m pip install . --ignore-installed --no-deps -vv
+  number: 1
 
 requirements:
   host:
-    - python
+    - python >=3
     - pip
   run:
-    - python
-    - biopython >=1.63
-    - intervaltree >=2.1.0
-    - joblib >=0.9.4
-    - pandas >=0.13.1
-    - pybedtools >=0.6.4
-    - pysam >=0.8.3
-    - r-base
-    - r-tidyverse
+    - python >=3
+    - nextgenmap=0.5.5
+    - samtools=1.10
+    - varscan=2.4.4
+    - r-tidyverse=1.3.0
+    - r-matrixstats=0.55.0
+    - r-gridextra=2.3
+    - r-getopt=1.20.3
+    - joblib=0.14.0
+    - pandas=0.25.3
+    - biopython=1.74
+    - pybedtools=0.8.0
+    - intervaltree=3.0.2
```

Comment:
Do we really need such exact pinning? For example, will samtools 1.9 really not work? This seems overly strict.

Reply:
I just created a conda env for testing and wrote that to an `environment.yml`. I kinda need it reproducible though (at least with regards to `nextgenmap` and `varscan`), so samtools we could relax to `samtools>=1.9` for sure.

Reply:
What about pandas, biopython and the other python dependencies?

Reply:
Basically, the more packages that we have that are strictly pinned as dependencies the more likely the recipe as a whole will become unmaintainable quickly.

Reply:
Those should be fine more relaxed as well - again just pulled out of the conda environment.